server/routes/movie.ts

```diff
@@ -130,14 +133,24 @@ movieRoutes.get('/:id/ratings', async (req, res, next) => {
       Number(movie.release_date.slice(0, 4))
     );
 
-    if (!rtratings) {
+    let imdbRatings;
+    if (movie.imdb_id) {
+      imdbRatings = await imdbApi.getMovieRatings(movie.imdb_id);
+    }
+
+    if (!rtratings && !imdbRatings) {
       return next({
         status: 404,
-        message: 'Rotten Tomatoes ratings not found.',
+        message: 'No ratings found.',
       });
     }
 
-    return res.status(200).json(rtratings);
+    const ratings: RatingResponse = {
+      ...(rtratings ? { rt: rtratings } : {}),
+      ...(imdbRatings ? { imdb: imdbRatings } : {}),
+    };
+
+    return res.status(200).json(ratings);
```

Comment:
This is a breaking API change. We should create a new endpoint for this newly returned object shape.

Reply:
@sct  How do you envision this change?

My line of thought is: since the `ratings` returned previously were coupled to rt and here the object returned is less specific and more true to the generic path`/:id/ratings`, this is more like a path correction of something that should have been `/:id/ratings/rottentomatoes`.

But it is certainly a breaking change.

Reply:
Can just name it something like `ratingscombined`. Just important we don't break the current endpoint.

Reply:
Got it, pushed a new comment about it. I didn't see a script to update the openapi definition, should I update it manually?

Reply:
Yes you will nee to update it manually